app/screens/zone_group_screen.ts

```diff
@@ -30,28 +30,28 @@ export default class ZoneGroupScreen extends Screen {
 
                 this.zoneGroups = msg.zoneGroups;
 
-                const VTList = document.getElementById("zone_group_list");
+                const VTList = document.getElementById("zone_group_list") as VirtualTileList;
 
                 const NUM_ELEMS = this.zoneGroups.length;
 
                 VTList.delegate = {
-                  getTileInfo: function(index: number): IMenuItem {
+                  getTileInfo: function(index) {
                     return {
                       type: "zone-group-pool",
                       value: "Menu item",
                       index: index
                     };
                   },
-                  configureTile: (function(tile: any, info: IMenuItem): void {
+                  configureTile: (tile, info) => {
```

Comment:
`tile` and `info` have types defined for the [`VirtualTileListDelegate`](https://github.com/SergioMorchon/fitbit-sdk-types/blob/master/device/document/elements.d.ts#L58-L61).